src/libraries/System.Net.Security/src/System/Net/Security/SslStreamCertificateContext.cs

```diff
@@ -8,8 +8,8 @@ namespace System.Net.Security
 {
     public partial class SslStreamCertificateContext
     {
-        internal readonly X509Certificate2 Certificate;
-        internal readonly X509Certificate2[] IntermediateCertificates;
+        public readonly X509Certificate2 Certificate;
+        public readonly X509Certificate2[] IntermediateCertificates;
```

Comment:
Would it make sense to add another comment here mentioning that these fields are not part of the ref assembly and are not, in fact, reachable in user code?